src/objprint/frame_analyzer.py

```diff
@@ -102,3 +102,27 @@ def _splitlines_no_ff(self, source: str) -> List[str]:  # pragma: no cover
         if next_line:
             lines.append(next_line)
         return lines
+
+    def return_object(self, frame: Optional[FrameType]) -> bool:
+        if frame is None:
+            return True
+        try:
+            current_frame: Optional[FrameType] = frame
+            while current_frame:
+                filename = current_frame.f_code.co_filename
+                if filename == "<stdin>" or filename == "<console>":
+                    return False
+                current_frame = current_frame.f_back
+        except AttributeError:  # pragma: no cover
+            pass  # pragma: no cover
```

Comment:
Where do you expect this `AttributeError`?

Reply:
Nowhere in specific, I will remove this try-except block.